lib/ansible/modules/web_infrastructure/jenkins_plugin.py

```diff
@@ -692,15 +692,16 @@ def _enabling(self, action):
 
         return changed
 
-    def _pm_query(self, action, msg):
+    def _pm_query(self, action, msg, http_method='GET'):
```

Comment:
I think that all plugin interaction under `/pluginManager/plugin` should be a `POST` request, not only `doUninstall` action (`pin`, `unpin`, `makeEnabled`, and `makeDisabled` should be `POST` requests too).